R/aaa_survival_prob.R

```diff
@@ -145,3 +145,142 @@ matrix_to_nested_tibbles_survival <- function(x, time) {
   dplyr::group_nest(res, .row, .key = ".pred")$.pred
 }
 
+
+
+# summary_survfit helpers -------------------------------------------------
+
+
+survfit_summary_typestable <- function(object){
+  # make matrix of dimension n_times x n_obs
+  sanitize_element <- function(x) {
+    if (!is.matrix(x)) {
+      x <- matrix(x, ncol = length(object$n))
+    }
+    x
+  }
+  # sanitize elements we care about
+  elements <- available_survfit_summary_elements(object)
```

Comment:
this is necessary because survfit objects from glmnet do not have standard errors and confidence intervals